mcenter_cli/parallelm/mcenter_cli/mcli.py

```diff
@@ -0,0 +1,100 @@
+import sys
+import logging
+
+from cliff.app import App
+from cliff.commandmanager import CommandManager
+
+from parallelm.mcenter_cli import version
+from parallelm.mcenter_client.mcenter_client import MCenterClient
+
+
+class MCli(App):
+
+    DEFAULT_LOG_LEVEL = "error"
```

Comment:
why is DEFAULT_LOG_LEVEL set to "error"? Thought that would be typically set "warning".

Reply:
Fixed